modules/claims_api/app/controllers/claims_api/base_form_controller.rb

```diff
@@ -20,6 +20,24 @@ def schema
 
     private
 
+    def verification_itf_expiration
+      unless itf_service.get_active('compensation')['intent_to_file'].expiration_date > Time.now.utc
```

Comment:
@bastosmichael what happens here if there is no current ITF for the target veteran?

Reply:
I went ahead and made the change to ensure it also fails if intent to file doesn't exist